### PR TITLE
Gp3 throughput and IOPS support

### PR DESCRIPTION
**Pull request by schmidtsv**
Adding support for both iops and throughput values of GP3 disks. 

- Added tests that test iops and throughput in the same fashion as io1s iops are tested
- Works on root, ephemeral and persistent disks.
- Note: values below the base of 3000 iops and 125mbps that are specified will be bumped by amazon to the baseline. 
- Test suite for block_device_manager are green. Due to me not having a playground AWS account I have not ran the full suite (as I did not touch most of the CPI)
- Manual integration test on AWS done.

### CODE REVIEW

**Comment by lnguyen** [comment]
Can you squash commit down @schmidtsv ?

**Comment by schmidtsv** [comment]
> Can you squash commit down @schmidtsv ?

I squashed it down

**Review by lnguyen — APPROVED** [review]
LGTM

File: src/bosh_aws_cpi/lib/cloud/aws/volume_properties.rb [hunk 1]
```diff
@@ -60,6 +63,14 @@ def root_disk_config
         if @type == 'io1' && @iops > 0
           root_device[:iops] = @iops
         end
+        #On GP3 you can leave those values empty and get 3000 iops/125mb/s throughput
+        if @type == 'gp3' && @iops != nil && @iops > 0
+          root_device[:iops] = @iops
+        end
```
Comment (rkoster) [inline src/bosh_aws_cpi/lib/cloud/aws/volume_properties.rb:69]:
```suggestion
        #On GP3 you can leave those values empty and get 3000 iops/125mb/s throughput
        if %w(io1 gp3).include?(@type) && @iops.to_i.nonzero?
          root_device[:iops] = @iops
        end
```

File: src/bosh_aws_cpi/lib/cloud/aws/volume_properties.rb [hunk 2]
```diff
@@ -60,6 +63,14 @@ def root_disk_config
...
+
+        if @type == 'gp3' && @throughput != nil && @throughput > 0
+          root_device[:throughput] = @throughput
+        end
```
Comment (rkoster) [inline src/bosh_aws_cpi/lib/cloud/aws/volume_properties.rb:73]:
```suggestion
        if @type == 'gp3' && @throughput.to_i.nonzero?
          root_device[:throughput] = @throughput
        end
```

**Review by rkoster — CHANGES_REQUESTED** [review]
Just some small suggestions, but looks good overal.

**Review by rkoster — APPROVED** [review]

**Comment by rkoster** [comment]
Thanks! @schmidtsv

**Pull request merged**